main.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <string>
+#include "Flowers.h"
+#include "Flowersshop.h"
```

Comment:
There is no need to include these two lines becouse they are included in Bouquet.h. Please remove them

Reply:
Fixed.